### DISCUSSION TITLE
does anybody know how to install two wiki.js instance on same server?

### DISCUSSION THREAD

**Discussion by b0risboris** [requarks/wiki#5078] [Help / Questions]
hi all, 
i want to install two wiki.js instances for two domains in the same server... how can i do this?..
for example,,,,  wiki.abc.com   and wiki.xyz.com in the same server... 
OS: Ubuntu 20LTS and Docker
Wiki.js version: 2.0.1
Database engine: PostgreSQL 9.6` 
server : nginx (i dont know apache)

**Comment by b0risboris** [comment]
IF NObody is here to answer users doubts, then why this forsaken forum is created??.... just shut it down and save others times mofos

**Reply by NGPixel** [reply]
Do you seriously expect an answer in less than 24h on an open source project? Users give their free time to contribute to it, so you can't possibly expect an answer right away and it's not a guarantee that someone will know the answer either. If that doesn't work for you, then feel free to buy a paid software with paid support. Also, how about being a bit more respectful in your posts?

Your question is very general and isn't specific to Wiki.js. There're many ways to achieve that... run 2 instances on different ports, use multiple docker containers, etc.

**Reply by monkey-of-the-hill** [reply]
I know last comment was over a year ago - but I'm trying to run two instances (containers on different ports each with their own db instance) and once I've got the second one up they both can't stay logged in for more than a few seconds. I've got them on separate docker networks to try and isolate them as much as possible. Even have database names different (even though they're on separate containers anyway). One by itself will stay logged in for hours - as soon as I spin up my "dev" instance I can't stay logged in for more than a few seconds. I think I might try different hosts and see if it still acts the same way. This is my method for "approvals" - dev instance pushes to a git dev branch - approver cherry picks or just pulls commits into the main branch which then syncs with the "production" instance.  Any other suggestions? or ideas?
